### PR TITLE
Add absolute dates to weather forecast

### PR DESCRIPTION
**Pull request by oraclesean**
- Does the pull request solve a **related** issue?
No

-  If so, can you reference the issue like this `Fixes #<issue_number>`? 
N/A

- What does the pull request accomplish? Use a list if needed.
Add a Boolean configuration option `absoluteDates` to `modules/default/weather` to allow weather forecast dates to be formatted as absolute (MON, TUE) or relative (TODAY, TOMORROW).

- If it includes major visual changes please add screenshots.
N/A

### CODE REVIEW

File: modules/default/weather/forecast.njk [hunk 1]
```diff
@@ -8,9 +8,9 @@
         {% set forecast = forecast.slice(0, numSteps) %}
         {% for f in forecast %}
             <tr {% if config.colored %}class="colored"{% endif %} {% if config.fade %}style="opacity: {{ currentStep | opacity(numSteps) }};"{% endif %}>
-                {% if (currentStep == 0) and config.ignoreToday == false %}
+                {% if (currentStep == 0) and config.ignoreToday == false and config.absoluteDates == false %}
```
Comment (rejas) [inline modules/default/weather/forecast.njk:11]: this config option should also be set in the defaults of the weather.js module

Comment (MichMich) [reply modules/default/weather/forecast.njk:11]: Indeed!

**Comment by MichMich** [comment]
Nice idea! @rejas made a valid point. 

Also: since this is a new feature, don't forget to send a PR to the docs repo after merge. :)

**Review by oraclesean — COMMENTED** [review]
Change made to weather.js, thanks for catching that!

**Comment by rejas** [comment]
Nice. Now it would be awesome if you could add a testcase for that. Think you can do it or do you need help?

**Comment by oraclesean** [comment]
@rejas yes I'd appreciate some guidance for a test case. JS is not my ninja area 😄

File: modules/default/weather/weather.js [hunk 2]
```diff
@@ -43,7 +43,8 @@ Module.register("weather", {
 		onlyTemp: false,
 		showPrecipitationAmount: false,
 		colored: false,
-		showFeelsLike: true
+		showFeelsLike: true,
+		absoluteDates: false
```
Comment (khassel) [inline modules/default/weather/weather.js:47]: if this is a property for the forecast, shouldn't this inserted in `weatherforecast.js` instead?

Comment (oraclesean) [reply modules/default/weather/weather.js:47]: I thought the current and forecast modules were deprecated in favor of the standalone weather module.

Comment (khassel) [reply modules/default/weather/weather.js:47]: ups, sorry for the noise, was irritated when looking in the tests (theoretically I should know this ...)

Comment (MichMich) [reply modules/default/weather/weather.js:47]: 😅👍🏻

**Comment by khassel** [comment]
wrote a test for this, will provide a PR when this is merged.

**Comment by MichMich** [comment]
Done!

**Pull request merged**